fairlearn/metrics/_multi_curve_plot.py

```diff
@@ -0,0 +1,126 @@
+from ..utils._input_validation import _validate_and_reformat_labels, _validate_and_reformat_labels_and_sf, check_consistent_length
+from ..postprocessing._plotting import _MATPLOTLIB_IMPORT_ERROR_MESSAGE
+from ._make_derived_metric import _DerivedMetric
+from typing import Callable, List, Union
+
+def plot_model_comparison(
+    *,
+    x_axis_metric: Callable[..., Union[float, int]],
+    y_axis_metric: Callable[..., Union[float, int]],
+    y_true,
+    y_preds,
```

Comment:
> 
>     * as `y_pred` we pass a dictionary with `{model_id: predictions}`, but nothing has been discussed with what to do with these `model_id`'s. Any suggestions? Should we add another parameter `label: bool` and if true print `model_id`-labels alongside datapoints?

Would be awesome if we could somehow make it clear which point corresponds to which model, otherwise it's hard to compare them. I can think of two ways: labels next to the plotted points, or color coding the points. There are problems with both methods depending on the use case, of course, but the labels are definitely preferable IMO.

Reply:
I usually prefer legends (with color code or symbols), as adding labels next to points can result in quite a crowded plot that's difficult to read - especially if there is only one default setting for distances, font, etc. I'm happy either way though.

Reply:
I think there's something to be said for either option, but I'd leave it up to the user to choose. We could provide a very crowded example with legend, and a less crowded one with labels if we want to go all out here. I will say color-coding also has its limits after about 8-12 colors since they start looking the same even to me (so I can only imagine how hard it is for colorblind folks to distinguish them). So ideally I'd let the user configure it and provide examples for how to configure it. @SeanMcCarren let me know if you don't know how to get these things set up. Happy to help!